akka-projection-core/src/main/scala/akka/projection/internal/InternalProjectionState.scala

```diff
@@ -45,6 +45,8 @@ private[akka] abstract class InternalProjectionState[Offset, Envelope](
     statusObserver: StatusObserver[Envelope],
     settings: ProjectionSettings) {
 
+  type Thingy = OpenEnvelope[Offset, Envelope]
```

Comment:
if we want to keep this alias at all it should be renamed, maybe `EnvelopeOffset` (and we could swap the order of the type parameters in all places since Envelope is the primary thing, and Offset is derived from it)

Reply:
I didn't mean to use a type alias at all. I'm deferring the naming until we deem it useful to follow this approach.